WrapImage/nifti_wrapper.py

```diff
@@ -2,9 +2,12 @@
 import json
 import os
 import nibabel as nib
+from src.wrappers.OsipiBase import OsipiBase
+from utilities.data_simulation.GenerateData import GenerateData
 import numpy as np
+import random
 
-def read_nifti_4d(input_file):
+def read_nifti_file(input_file):
```

Comment:
This should also return the header or just the complete nifti for the user to separate.